qdrant_client/qdrant_remote.py

```
@@ -321,6 +321,18 @@ def close(self, grpc_grace: float | None = None, **kwargs: Any) -> None:
         self._closed = True
 
+    def server_info(self) -> types.VersionInfo | None:
+        """Return the Qdrant server's full ``VersionInfo`` model (with
+        ``title``, ``version``, and ``commit``). Calls the REST ``/``
+        endpoint via the existing service API binding.
+
+        On any failure (timeout, connection refused, non-2xx, API
+        error), returns ``None`` rather than raising."""
+        try:
+            return self.openapi_client.service_api.root()
+        except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)